runtime/lib/ttnn/program.cpp

```diff
@@ -400,6 +404,42 @@ run(::tt::target::ttnn::MatmulOp const *op, ::ttnn::Device &device,
 }
 // ANCHOR_END: adding_an_op_matmul_runtime
 
+static void
+run(::tt::target::ttnn::Conv2dOp const *op, ::ttnn::Device &device,
+    std::unordered_map<std::uint32_t, ::ttnn::Tensor *> &liveTensors,
+    std::list<::ttnn::Tensor> &tensorPool) {
+  auto &input = *liveTensors.at(op->in0()->global_id());
+  auto &weight = *liveTensors.at(op->in1()->global_id());
+  auto config = ::ttnn::operations::conv2d::Conv2dConfig();
+  config.dtype = input.dtype();
+  config.weights_dtype = weight.dtype();
+  ::ttnn::Tensor ans;
```

Comment:
I think `out` is more in line with the naming in flatbuffer and other places in runtime.